airbyte-integrations/connectors/destination-jdbc/src/test/java/io/airbyte/integrations/destination/jdbc/TestJdbcSqlOperations.java

```diff
@@ -19,4 +24,17 @@ public void insertRecordsInternal(final JdbcDatabase database,
     // Not required for the testing
   }
 
+  @Test
+  public void testCreateSchemaIfNotExists() {
+    final JdbcDatabase db = Mockito.mock(JdbcDatabase.class);
+    final var schemaName = "foo";
+    try {
+      Mockito.doThrow(new SQLException("TEST")).when(db).execute(Mockito.anyString());
+      createSchemaIfNotExists(db, schemaName);
```

Comment:
I think there's an `assertThrows` method for this exact situation - currently your test will pass even if this method doesn't throw

Reply:
Ahh good catch! that's what I was trying to remember